sdks/python/apache_beam/io/gcp/tests/bigquery_matcher.py

```diff
@@ -130,7 +130,7 @@ def __init__(self, project, query, data):
 
     self.project = project
     self.query = query
-    self.expected_data = [sorted(i) for i in data]
```

Comment:
The current behavior of the bigquery_matcher is to compare data retrieved from bigquery by sorting both the list of rows (L146) but also the values in a row.  This gave an error in python 3 `TypeError: unorderable types` when multiple types were present in a row e.g. when `data` is `('a',1)`. I removed the sorting because I think it does not make sense to order the values within a bigquery row